src/main/java/uk/gov/hmcts/reform/notificationservice/service/NotificationService.java

```diff
@@ -43,6 +43,7 @@ public void processPendingNotifications() {
         int postponedCount = 0;
 
         for (var notification : notifications) {
+            // DO NOT CHANGE, used in alert
             log.info("Sending error notification. {}", notification.basicInfo());
```

Comment:
Given there's a story about adding a delay between receiving  message on the queu and sending the error notification, the alert should be based on the former event.